app/api/tracks.py

```diff
@@ -0,0 +1,76 @@
+from flask_rest_jsonapi import ResourceDetail, ResourceList, ResourceRelationship
+from marshmallow_jsonapi.flask import Schema, Relationship
+from marshmallow_jsonapi import fields
+from app.models import db
+from app.models.track import Track
+from app.api.helpers.permissions import jwt_required
+
+
+class TrackSchema(Schema):
+    """
+    Api Schema for track model
+    """
+    class Meta:
+        """
+        Meta class for User Api Schema
+        """
+        type_ = 'track'
+        self_view = 'v1.track_detail'
+        self_view_kwargs = {'id': '<id>'}
+        self_view_many = 'v1.track_list'
+
+    id = fields.Str(dump_only=True)
+    name = fields.Str(required=True)
+    description = fields.Str()
+    track_image_url = fields.Str()
+    color = fields.Str()
+    location = fields.Str()
+    event = Relationship(attribute='event',
+                         self_view='v1.track_event',
+                         self_view_kwargs={'id': '<id>'},
+                         related_view='v1.event_detail',
+                         related_view_kwargs={'id': '<id>'},
+                         schema='EventSchema',
+                         type_='event')
+
+    sessions = Relationship(attribute='sessions',
+                            self_view='v1.track_sessions',
+                            self_view_kwargs={'id': '<id>'},
+                            related_view='v1.session_list',
+                            related_view_kwargs={'track_id': '<id>'},
+                            schema='SessionSchema',
+                            many=True,
+                            type_='session')
+
+
+class TrackList(ResourceList):
+    """
+    List and create Tracks
+    """
+    decorators = (jwt_required, )
+    schema = TrackSchema
+    data_layer = {'session': db.session,
+                  'model': Track,
+                  }
+
+
+class TrackDetail(ResourceDetail):
+    """
+    Track detail by id
+    """
```

Comment:
As mentioned above, there should be way to filter track's id from session's id which will passed when ```/sessions/<int:id>/track```